scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -604,8 +656,14 @@ public void statusChanged(SchedulerStatus status) {
                 schedPauseButton.setEnabled(false);
                 schedResumeButton.setEnabled(false);
                 schedKillButton.setEnabled(true);
+                schedulerStatusLabel.setIcon(SchedulerImages.instance.scheduler_stop_16().getSafeUri().asString());
+                break;
+            default:
+                LOGGER.warning("Unexpected scheduler status");
                 break;
         }
+        // Update the scheduler status label
+        schedulerStatusLabel.setContents("Status:" + status.name());
```

Comment:
You can add a space after the ':' to comply to english punctuation.

Reply:
Sorry but Denis approved without the space..